src/documents/InclusiveMakingAndHacking.html.md.handlebars

```diff
@@ -0,0 +1,205 @@
++---
+title: Inclusive Making and Hacking
+layout: default
+category: Methods
+---
+
+Hackathons, makeathons and similar events have become very popular in recent years, both in traditional learning contexts such as schools, colleges and universities, and in broader world.
+
+Ideas of "hacking" and "making" have also been popularized as approaches to creatively and collaboratively solve problems in society, business and government.
+
+Where does Inclusive Design fit into these approaches?
+
+## On Terminology
+
+### Hackathon
+
+We use the term "hackathon" expansively in this entry to refer to events that emphasize intensive short-term collaboration between small groups. The groups typically ideate, design and prototype around themes such as <a href="http://arthackathon.co.uk/" target="_blank" class="link-external">art</a> or <a href="http://civictech.ca/2016/04/08/codeacross-toronto-community-collaboration-in-celebration-of-international-open-data-day/" target="_blank" class="link-external">open data</a>, then present their result in some form to to the larger group throughout the event or at the end.
+
+While these events are often focused on producing or proposing new software or hardware as a means of addressing a social problem or creating a new product, this is not stricly necessary; hackathons have been held to produce
+<a href="https://www.policyhack.com.au/" target="_blank" class="link-external">government policy</a>, or ideas to address <a href="http://foodinnovationprogram.org/hacking-hunger/" target="_blank" class="link-external">food security</a>.
+
+In learning settings hackathons might be held as part of course curriculum or as an extracurricular activity for interested students. Academic institutions might seek to partner with community groups for hackathons to encourage students and faculty to form connections outside the institution.
```

Comment:
Missing comma between "In learning settings" and "hackathons" in the first sentence.